plugins/techdocs/src/home/components/Tables/DocsTable.tsx

```diff
@@ -45,6 +45,7 @@ export type DocsTableProps = {
   entities: Entity[] | undefined;
   title?: string | undefined;
   loading?: boolean | undefined;
+  options?: object | undefined;
```

Comment:
Could you create a specific TableOptions type to help people understand what properties can be set on this object?